sensory_patch.py

```
@@ -0,0 +1,88 @@
+import sys, re
+
+def patch_main_scene():
+    path = 'src/scripts/scenes/MainScene.js'
+    with open(path, 'r') as f: content = f.read()
```

Ruff (0.15.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)